test/reaper-ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: reaper
+  namespace: default
+spec:
+  rules:
+    - http:
+        paths:
+          - backend:
+              serviceName: reaper-nodeport
+              servicePort: 8080
+            path: /webui/*
+            pathType: ImplementationSpecific
+status:
+  loadBalancer:
+    ingress:
+      - ip: 34.117.48.220
```

Comment:
It's surprising to see a static IP here. Is this an allocated IP that will be the same between runs?

Reply:
@chaitu6022 can we please remove the ip?

Reply:
yes, we can remove, missed this comment.